src/main/resources/modules/CMS130v10.json

```diff
@@ -0,0 +1,281 @@
+{
+  "name": "CMS130",
+  "remarks": [
+    "Synthea module based on CMS130v10 - Colorectal Cancer Screening"
+  ],
+  "states": {
+    "Initial": {
+      "type": "Initial",
+      "direct_transition": "IPP_Age_Guard",
+      "name": "Initial"
+    },
+    "IPP_Age_Guard": {
+      "type": "Guard",
+      "allow": {
+        "condition_type": "And",
+        "conditions": [
+          {
+            "condition_type": "Age",
+            "operator": ">=",
+            "quantity": 50,
+            "unit": "years",
+            "value": 0
+          },
+          {
+            "condition_type": "Age",
+            "operator": "<=",
+            "quantity": 75,
+            "unit": "years",
+            "value": 0
+          }
+        ]
+      },
+      "direct_transition": "Measurement_Period_Guard",
+      "name": "IPP_Age_Guard"
+    },
+    "Terminal": {
+      "type": "Terminal",
+      "name": "Terminal"
+    },
+    "Measurement_Period_Guard": {
+      "type": "Guard",
+      "allow": {
+        "condition_type": "Date",
+        "operator": ">=",
+        "year": 2022,
+        "value": 0
+      },
+      "name": "Measurement_Period_Guard",
+      "distributed_transition": [
+        {
+          "transition": "Telehealth_Visit",
+          "distribution": 0.2
+        },
+        {
+          "transition": "Qualifying_Encounter",
+          "distribution": 0.7
+        },
+        {
+          "transition": "Check_For_Procedure",
+          "distribution": 0.1
+        }
+      ]
+    },
+    "Qualifying_Encounter": {
+      "type": "Encounter",
+      "telemedicine_possibility": "none",
+      "name": "Qualifying_Encounter",
+      "direct_transition": "Encounter_Occurred",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 444971000124105,
+          "display": "Annual Wellness Visit",
+          "value_set": "2.16.840.1.113883.3.526.3.1240"
+        },
+        {
+          "system": "SNOMED-CT",
+          "code": 185460008,
+          "display": "Home Healthcare Services",
+          "value_set": "2.16.840.1.113883.3.464.1003.101.12.1016"
+        },
+        {
+          "system": "SNOMED-CT",
+          "code": 185463005,
+          "display": "Office Visit",
+          "value_set": "2.16.840.1.113883.3.464.1003.101.12.1001"
+        },
+        {
+          "system": "CPT",
+          "code": "99395",
+          "display": "Preventive Care Services - Established Office Visit, 18 and Up",
+          "value_set": "2.16.840.1.113883.3.464.1003.101.12.1025"
+        },
+        {
+          "system": "CPT",
+          "code": 99385,
+          "display": "Preventive Care Services-Initial Office Visit, 18 and Up",
+          "value_set": "2.16.840.1.113883.3.464.1003.101.12.1023"
+        }
```

Comment:
Unless you really need to exercise all options from the measure I'd suggest removing these - licensing is a question and I don't know if this small set constitutes fair use.